src/crypto.cpp

```diff
@@ -10,6 +11,9 @@ using hpke::KDF;       // NOLINT(misc-unused-using-decls)
 using hpke::KEM;       // NOLINT(misc-unused-using-decls)
 using hpke::Signature; // NOLINT(misc-unused-using-decls)
 
+using mls::log::Log;
+static const auto log_mod = "crypto"s;
+
```

Comment:
why do we need log_mod to be a string_literal instead of normal string

Reply:
I think it could also be a std::string, this is just syntactic sugar.